scripts/terraform/terraform.mk

```diff
@@ -55,23 +58,6 @@ terraform-shellscript-lint: # Lint all Terraform module shell scripts @Quality
 		file=$${file} scripts/shellscript-linter.sh
 	done
 
-# ==============================================================================
```

Comment:
Can we also remove along with the example the following, please:

```
	terraform-example-clean \
	terraform-example-destroy-aws-infrastructure \
	terraform-example-provision-aws-infrastructure \
```

This is in the `${VERBOSE}.SILENT` section at the bottom of the file.

Reply:
Good catch - fixed.